include/utils.hpp

```diff
@@ -1,18 +1,28 @@
 #ifndef OBC_UTILS_HPP
 #define OBC_UTILS_HPP
 
+#include <Arduino.h>
+
 #include <type_traits>
 
 namespace utl {
 
 enum class Logger { Debug, Flight };
 
+enum class Modes { Idle, Measurements, Experiment, Uplink };
```

Comment:
Czym różni się Measurements od Experiment? Chyba czuje, różnice, ale czy to oznacza, że gdy odpalimy eksperyment to nie wykonujemy pomiarów z samego obc? (w sensie, czy mode Measurements pokrywa się jakoś z Experiment zadaniami). 

Ogólnie to szczegół, przy takiej implementacji chyba łatwo będzie zedytować te stany/zdefiniować nowe. O to chodziło.